### PR TITLE
DASHBASE-282 Converter any value

### PR DESCRIPTION
**Pull request by XSAM**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -198,6 +198,13 @@
             <artifactId>commons-text</artifactId>
             <version>1.9</version>
         </dependency>
+
+        <dependency>
+            <groupId>io.confluent</groupId>
+            <artifactId>kafka-connect-avro-converter</artifactId>
+            <version>6.1.1</version>
```
Comment (sincejune) [inline pom.xml:205]: The dependency is already declared.

File: src/main/java/io/dashbase/connect/JsonSerializer.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
+package io.dashbase.connect;
```
Comment (sincejune) [inline src/main/java/io/dashbase/connect/JsonSerializer.java:1]: If this is no changes there, would you please try to use `org.apache.kafka.connect.json.JsonSerializer`?

File: src/test/java/logs/AnyValue.java [hunk 3]
```diff
@@ -0,0 +1,282 @@
+/**
```
Comment (sincejune) [inline src/test/java/logs/AnyValue.java:1]: I prefer to use Avro maven plugin to generate these files automatically. ref: https://avro.apache.org/docs/1.10.2/gettingstartedjava.html

**Review by sincejune — COMMENTED** [review]
Actually, we need to know its performance compared with the base branch. Would you take the work before it get merged?

File: pom.xml (hunk 1 above)
Comment (XSAM) [reply pom.xml]: Done

File: src/test/java/logs/AnyValue.java (hunk 3 above)
Comment (XSAM) [reply src/test/java/logs/AnyValue.java]: Done

**Comment by sincejune** [comment]
Talked with @XSAM offline, the performance shouldn't be bad cuz this PR introduced a new JSON converter which did similar things as the old one.

**Review by sincejune — APPROVED** [review]

**Pull request merged**